dist/templates/ovnkube-webhook.yaml.j2

```diff
@@ -43,3 +43,24 @@ spec:
     - port: 443
       targetPort: webhook-api
 ---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: ovnkube-admission-webhook
+webhooks:
+  - name: ovnkube-webhook.ovn-kubernetes.svc
+    clientConfig:
+      service:
+        name: ovnkube-webhook
+        namespace: ovn-kubernetes
+        path: "/egressip"
+      caBundle: "{{ ca_pem_64 }}"
+    rules:
+      - operations: [ "CREATE" ]
+        apiGroups: ["k8s.ovn.org"]
+        apiVersions: ["v1"]
+        resources: ["egressip", "egressips", "eip"]
+        scope: "Cluster"
+    admissionReviewVersions: ["v1", "v1beta1"]
```

Comment:
This says that your admission hook can work against an apiserver implementing either the v1 or v1beta1 API, but your hook always returns a `v1.AdmissionReview`, so that's a lie.

We don't need to support the v1beta1 API anyway.

Reply:
You're right, this is from one month ago when the CRDs had `v1beta1` defined. Will remove